tests/test_inputlocationmessagecontent.py

```diff
@@ -19,46 +19,22 @@
 
 import pytest
 
-from telegram import InputMessageContent, InputLocationMessageContent
-
-
-@pytest.fixture(scope='function')
-def json_dict():
-    return {
-        'longitude': TestInputLocationMessageContent.longitude,
-        'latitude': TestInputLocationMessageContent.latitude,
-    }
+from telegram import InputLocationMessageContent
 
 
 @pytest.fixture(scope='class')
 def input_location_message_content():
-    return InputLocationMessageContent(TestInputLocationMessageContent.longitude,
-                                       TestInputLocationMessageContent.latitude)
+    return InputLocationMessageContent(longitude=TestInputLocationMessageContent.longitude,
```

Comment:
because they were in the wrong order and adding kwargs was lazier then reversing the order. Will fix though